tests/conftest.py

```diff
@@ -1154,75 +1154,87 @@ def cluster_load(
     io_in_bg = config.RUN.get('io_in_bg')
     log_utilization = config.RUN.get('log_utilization')
     io_load = config.RUN.get('io_load')
+    cluster_load_error = None
 
     # IO load should not happen during deployment
-    deployment_test = True if 'deployment' in request.node.items[0].location[0] else False
-    if io_in_bg and not deployment_test:
-        io_load = int(io_load) * 0.01
-        log.info(wrap_msg("Tests will be running while IO is in the background"))
-        log.info(
-            "Start running IO in the background. The amount of IO that "
-            "will be written is going to be determined by the cluster "
-            "capabilities according to its limit"
-        )
-        cl_load_obj = ClusterLoad(
-            project_factory=project_factory_session,
-            sa_factory=service_account_factory_session,
-            pvc_factory=pvc_factory_session,
-            pod_factory=pod_factory_session,
-            target_percentage=io_load
+    try:
```

Comment:
we don't need to wrap the entire implementation with try except. The only call needed to be wrapped is `cl_load_obj.reach_cluster_load_percentage()`

Reply:
@ebenahar I am not sure what all calls do we do in init of: ClusterLoad , 
I saw there is initialization of prometheusAPI and some others factories so I for sure wrapped everything as if the code will fail somewhere else this is more safe option that we will not fail the rest of the tests cause of the issue here.

So I can end the except at line  1186.

As this autouse session scope fixture is very sensitive and each error/exception means whole execution will fail - and also found out that in case of tier1 it's failing each test case and running must gather for it. So it waste a lot of time.